src/Nri/Ui/SegmentedControl/V6.elm

```diff
@@ -0,0 +1,144 @@
+module Nri.Ui.SegmentedControl.V6 exposing (Config, Icon, Option, Width(..), view)
+
+{-|
+
+@docs Config, Icon, Option, Width, view
+
+-}
+
+import Accessibility.Styled exposing (..)
+import Accessibility.Styled.Role as Role
+import Css exposing (..)
+import Css.Foreign exposing (Snippet, adjacentSiblings, children, class, descendants, each, everything, media, selector, withClass)
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes as Attr exposing (css)
+import Html.Styled.Events as Events
+import Nri.Ui.Colors.Extra exposing (withAlpha)
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.CssFlexBoxWithVendorPrefix as FlexBox
```

Comment:
❓ Do we still need to vendor prefix? I never know

Reply:
No, not for flexbox